privacy/scrubber.go

```diff
@@ -83,6 +83,15 @@ func NewScrubber() Scrubber {
 }
 
 func (scrubber) ScrubRequest(bidRequest *openrtb2.BidRequest, enforcement Enforcement) *openrtb2.BidRequest {
+	var userExtParsed map[string]json.RawMessage
```

Comment:
Did you look into if you could push the request wrapper down to this level? There was a PR to clone the request wrapper in order to allow copies for each bidder, rather than just copying the base bid request. I don't recall if it got any further than that.